aws-sagemaker-python/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pulumi>=3.0.0,<4.0.0
+pulumi-aws>=5.0.0,<6.0.0
```

Comment:
Any reason _not_ to use AWS v6 here? Does it not work?

Reply:
we determined on zoom that changes to requirements.txt was unnecessary

Reply:
For `boto3` (required to use the test script), correct. For now, let's leave this as is, and we'll take a follow up item to run this with AWS v6 and update the template as needed.